.github/workflows/jekyll.yml

```diff
@@ -4,24 +4,30 @@ on:
   push:
     branches: [main]
 
+permissions:
+  contents: read
+
 jobs:
   build_jekyll:
     runs-on: ubuntu-latest
     if: github.repository_owner == 'music-encoding'
 
+    permissions:
+      contents: write
+    
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
As written in [the recommendations](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) using shas applies to **third-party** actions. I would propose to stay here always with the latest main version.

Reply:
Indeed. I would regard any action that we do not maintain/own ourselves as third-party action here.  (BTW: v4 is pointing to 4.1.4 itself, but as targeting a tag instead of commit is much more vulnerable to tag mutation.)

Reply:
Alright.